apps/passport-client/components/screens/ProveScreen/GenericProveScreen.tsx

```diff
@@ -24,6 +24,27 @@ const Header = styled.div`
   align-items: center;
 `;
 
+/**
+ * Throws an error that indicates that the current window does not have an
+ * opener, and that the user should try again using a different browser.
+ *
+ * If the user requests a proof to be returned via `postMessage`, then the
+ * current window should have an opener - the window that opened the Zupass
+ * popup. However, some browsers seem not to set the `opener` property, so
+ * we need to check for that and throw an error if it's not set.
+ *
+ * To date, this seems to occur in in-app browsers like Metamask. However, it
+ * has also been reported on Chrome, although we're not yet sure what browser
+ * settings/extensions may be causing it.
+ *
+ * See "onProveCallback" below for more context.
+ */
+function throwNoOpenerError(): never {
+  throw new Error(
+    "Zupass was unable to send your proof to the opening window. This may be because your browser settings do not allow this, or because your browser does not support messaging between tabs."
```

Comment:
Let's add a call to action here. As a user, I wouldn't quite understand what to do here, and the details don't really help.

```suggestion
    "Zupass was unable to send your proof. Please retry this on a different browser or device."
```